docs/changelog.md

```diff
@@ -11,6 +11,7 @@
     * Host forwarding now supports multiple forwarding rules per app to the same container port for HTTP forwarding
     * Fixed a bug where intermediate containers were not removed when creating testing images
     * Revived a previous networking fix that was removed after Dusty moved to Docker Machine. This fix improves external networking performance by up to 50% for large file transfers.
+    * Fixed a bug which allowed a directory to be written to in HFS but did not allow that same directory to be pulled using NFS.
```

Comment:
Would word this more generally, just mention fixing some case-sensitivity bug or something